pynvim/util.py

```diff
@@ -40,3 +41,22 @@ def get_client_info(kind, type_, method_spec):
 
 
 VERSION = Version(major=0, minor=3, patch=2, prerelease='')
+
+
+def transform_keyerror(f):
+    """Decorator to transform "Key not found" NvimErrors into KeyErrors."""
```

Comment:
Also handles "Invalid option name" by now.
I.e. doc should be updated / generalized.

Might make sense also to have different wrappers for vars and options (for clarity).